CoreGraphUtilities/src/au/gov/asd/tac/constellation/graph/utilities/hashmod/Hashmod.java

```diff
@@ -74,7 +75,7 @@ public void setCSVFileStr(final String csvFileStr) {
     }
 
     public String[] getCSVFileHeaders() {
-        if (CollectionUtils.isNotEmpty(data)) {
+        if (data != null && !data.isEmpty()) {
```

Comment:
@CrucisGamma did the CollectionUtils not work as expected? If so what was the issue?